app/store/qiita.ts

```diff
@@ -429,15 +429,15 @@ export const actions: DefineActions<
         fetchCategorizedStockRequest
       )
 
-      const categorizedStocks: CategorizedStock[] = []
-      for (const stock of fetchCategorizedStockResponse.stocks) {
+      let categorizedStocks: CategorizedStock[] = []
+      fetchCategorizedStockResponse.stocks.map(stock => {
         const date: string[] = stock.article_created_at.split(' ')
         stock.article_created_at = date[0]
         const categorizedStock: CategorizedStock = Object.assign(stock, {
           isChecked: false
         })
-        categorizedStocks.push(categorizedStock)
-      }
+        categorizedStocks = [...categorizedStocks, categorizedStock]
+      })
```

Comment:
```suggestion
      const categorizedStocks = fetchCategorizedStockResponse.stocks.map(
        stock => {
          const date: string[] = stock.article_created_at.split(' ')
          stock.article_created_at = date[0]

          const categorizedStock: CategorizedStock = Object.assign(stock, {
            isChecked: false
          })

          return categorizedStock
        }
      )
```

まだリファクタリングの途中だと思うんだけど、ここをmapに置き換える場合、こうするのはどうだろう🐱？

`map` は新しい配列を作り出す効果があるから、こうすると `let` で宣言してる変数も消せるから良いと思う！

基本方針としては、基本的に配列操作は map, filter, reduce等を使う、ただしループの目的が新しい配列を作る事ではなく単純にループするだけのケースの場合は `for` を使うで良いと思う！

この記事が参考になるよ👌

https://qiita.com/rexiaxm7/items/f7d0c5e70ca3f1c5bbec

Reply:
なるほど！確かに`let`で宣言する必要はないね！
下記の通り、2箇所修正、修正したよ！
https://github.com/nekochans/qiita-stocker-nuxt/pull/87/commits/d0e785b48ef8b60dd43da05d6faff3dce3b962a2